acceptanceTests/tap_test.go

```diff
@@ -66,21 +66,12 @@ func TestTap(t *testing.T) {
 			entriesCheckFunc := func() error {
 				timestamp := time.Now().UnixNano() / int64(time.Millisecond)
 
-				entriesUrl := fmt.Sprintf("%v/entries?limit=%v&operator=lt&timestamp=%v", apiServerUrl, entriesCount, timestamp)
-				requestResult, requestErr := executeHttpGetRequest(entriesUrl)
-				if requestErr != nil {
-					return fmt.Errorf("failed to get entries, err: %v", requestErr)
-				}
-
-				entries := requestResult.([]interface{})
-				if len(entries) == 0 {
-					return fmt.Errorf("unexpected entries result - Expected more than 0 entries")
-				}
-
-				entry := entries[0].(map[string]interface{})
+				entries := getDBEntries(t, timestamp, entriesCount, 1*time.Second)
+				assertEntriesAtLeast(t, entries, 1)
```

Comment:
you need to stop the function if len(entries) == 0, so you won't continue if you know that you don't have entries

Reply:
Doesn't that cause code duplication? How about using `t.Fatalf` instead of `t.Errorf`?

Reply:
I guess not because it's tried several times. I'm not sure why there are `retriesExecute` everywhere but anyway. Let me make it return instead.

Reply:
I wanted to switch from t.Errorf to t.Fatalf for a long time, i'll do it next spring so let's leave it like that for now.
retriesExecute are called because it can take a few sec for the entries to get inserted into the db, and we don't want to fail the test if they didn't exist on the first check.

Reply:
Done with https://github.com/up9inc/mizu/commit/4879346f8fe06c68a8e5b477564d84e6ade8898f